packages/lesswrong/server/languageModels/autoTagCallbacks.ts

```diff
@@ -133,24 +131,17 @@ function preprocessHtml(html: string): string {
   return $.html();
 }
 
-export async function postToPrompt({template, post, promptSuffix, postBodyCache, markdownBody}: {
+export async function postToPrompt({template, post, promptSuffix, postBodyCache}: {
```

Comment:
I previously made some changes to this function to work with the `PostIsCriticismRequest`, this basically undoes that